ql/src/test/queries/clientpositive/like_control_characters.q

```diff
@@ -0,0 +1,15 @@
+set hive.mapred.mode=nonstrict;
+set hive.explain.user=false;
+set hive.vectorized.execution.enabled=true;
+
+create temporary table foo (col string) stored as orc;
+create temporary table bar (col binary) stored as orc;
+
+-- SORT_QUERY_RESULTS
+
+INSERT INTO bar values(unhex('6161-16161'));
+INSERT INTO foo SELECT col FROM bar;
+
+explain select col, count(*) from foo where col like '%bc%' group by col;
+select col, count(*) from foo where col like '%bc%' group by col;
```

Comment:
I still couldn't reproduce the exception as you described in [HIVE-26713](https://issues.apache.org/jira/browse/HIVE-26713) on master branch.
Did you try to test on the master branch?

Reply:
@zhangbutao I noticed that the query I tested and the query in the qfile were different and fixed it. I ran it on master, but before the fix, an [HIVE-26713](https://issues.apache.org/jira/browse/HIVE-26713) exception occurs.